### ISSUE TITLE
Can't Pair

### ISSUE THREAD

**Issue by dreyln** [heathbar/vizio-smart-cast#1]
Greetings,
I can't seem to get the pairing to work.  Logging the power state works just fine following your example and will return a 0 or a 1, but I get the following error when trying to run the pairing script:
(node:28804) UnhandledPromiseRejectionWarning: Unhandled promise rejection (rejection id: 1): TypeError: Cannot read property 'RESULT' of undefined
Basically, I get the Enter PIN: prompt but the TV never actually shows a PIN and then the script throws the above error no matter what.
I've got a vizio p65-c1 running 3.2.8.3.

**Comment by heathbar** [comment]
I'm wondering if Vizio didn't push an update. I'm also on 3.2.8.3 and I'm also having some trouble paring. I have pushed what I believe to be a fix in version 1.0.8 to npm. If it is still not working, try logging the error so I can better help. 

FWIW: I was seeing a lot of 'BLOCKED' errors which means that a pairing request was currently in progress. I didn't find any way around this other than to turn the TV off for several hours. (It may not have required that much time, but that's what I did.)

```
let smartcast = require('vizio-smart-cast');
let readline = require('readline'); // user input via cmd line
let tv = new smartcast('192.168.0.101');

// configure cmd line input
const rl = readline.createInterface({
    input: process.stdin,
    output: process.stdout
});

// Initiate a pairing request with a smartcast device
tv.pairing.initiate().then((response) => {

    // prompt the user for the pin that is displayed on the smartcast device
    rl.question('Enter PIN:', (answer) => {
        
        // send the pin to the smartcast device to complete the pairing process
        tv.pairing.pair(answer).then((response) => {

            // log the token to be used for future, authenticated requests
            console.log(response.ITEM.AUTH_TOKEN);
        });

        rl.close();
    });
}).catch((error) => {
    console.log('Error: ', error);
});
```

Lastly, if you are working on something for personal use, you could try using curl to get your auth token. 

`curl -k -H "Content-Type: application/json" -X PUT -d '{"DEVICE_ID":"12345","DEVICE_NAME":"cURL Example"}' https://myVizioTV:9000/pairing/start`

`curl -k -H "Content-Type: application/json" -X PUT -d '{"DEVICE_ID": "12345","CHALLENGE_TYPE": 1,"RESPONSE_VALUE": "1234","PAIRING_REQ_TOKEN": 0}' https://myVizioTV:9000/pairing/pair`

credit: https://github.com/exiva/Vizio_SmartCast_API

**Comment by dreyln** [comment]
Thanks for the response!

I was able to pair and get an auth token using curl.  I haven't tested the updated library yet.
